amundsen_application/api/search/v0.py

```diff
@@ -152,7 +161,15 @@ def _search(*, search_term: str, page_index: int) -> Dict[str, Any]:
                 'last_updated',
             ]
             results = response.json().get('results')
-            results_dict['results'] = [{key: result.get(key, None) for key in params} for result in results]
+            for result in results:
+                result['type'] = 'table'
+
+            tables = {
+                'total_results': response.json().get('total_results'),
+                'page_index': int(page_index),
+                'results': [{key: result.get(key, None) for key in params} for result in results],
```

Comment:
No need to have the extra loop here as we are already looping over the `results` on line 164. This can go under that loop. 
Or even, if the results are in `results_dict['tables']`, that means that the type is "table". Why do you need to specify that in each item instead? 
Same goes for the popular_tables API, you know these are tables, and can simply add the type "table" in the top level response instead of adding the type in each item. 

Not sure if I am missing something here.

Reply:
I will move the `type` assignment into the loop. I'm a complete python newbie so I'm not 100% sure how line 170 works but will find out.

We have plans for search to have an "all" category that will return ordered results of different types. Ranking among different search types will take time to get right, so our initial version will simply separate each type into different tabs. We also have a bookmark feature that will let users save tables, dashboards, and users. In these cases it will be very useful for each result to indicate its type.